src/main/java/commands/AddCommand.java

```diff
@@ -0,0 +1,19 @@
+package commands;
+
+import core.Ui;
+import data.Storage;
+import tasks.TaskList;
+
+//CHECKSTYLE.OFF: MissingJavadocType
```

Comment:
Should there be a descriptive header comment for the public class here?

Reply:
Yes sir absolutely! I was lazy and therefore did not want to include header for all the command classes.